### PR TITLE
Feature/ Subscriber comming soon / Onboarding Page

### PR DESCRIPTION
**Pull request by joartola**
Refactored some UI to make it reusable.
Created new route:
`datasubscriber/onboarding/notconnected`

### CODE REVIEW

File: webapp/pages/datasubscriber/onboarding/not-connected.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+              mb={10}
+            />
+            <OnBoardingContentPiece
+              textAlign="start"
+              title="Preserve privacy"
+              content="Pool your anonymized transaction data with other DAO members"
+            />
+          </Box>
+          <Box flex="1">
+            {/* TODO: Add svg component. KEVIN Which is the best way to do this in REACT? */}
```
Comment (kgelpes) [inline webapp/pages/datasubscriber/onboarding/not-connected.tsx:67]: @joartola  If it's an image we can use nextjs Image component with the svg as the src. If it's an icon you can copy and paste the svg code into a new component like this:

```
const SvgIcon = () => (<svg 
  xmlns="http://www.w3.org/2000/svg"
  className="h-6 w-6"
  fill="none"
  viewBox="0 0 24 24"
  stroke="currentColor"
  strokeWidth={2}
>
  <path 
  strokeLinecap="round"
  strokeLinejoin="round"
  d="M11 19l-7-7 7-7m8 14l-7-7 7-7"
/>
</svg>)
```

The advantage of using the 2nd option is that you could pass props like strokeWidth, color, etc and customize the svg.

**Review by kgelpes — APPROVED** [review]

**Comment by karensheng** [comment]
@joartola There was a typo in the Figma file and the correction is not reflected in the UI. It should be "Coming soon" on the subscriber landing page.

**Comment by joartola** [comment]
> @joartola There was a typo in the Figma file and the correction is not reflected in the UI. It should be "Coming soon" on the subscriber landing page.

image
Is it not like this?

**Comment by karensheng** [comment]
@joartola  It is spelled as "Comming" on the landing page right, which is incorrect. It should be "Coming".

**Pull request merged**